pkg/config/config.go

```diff
@@ -754,6 +754,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.socket", "/opt/datadog-agent/run/runtime-security.sock")
 	config.BindEnvAndSetDefault("runtime_security_config.enable_kernel_filters", true)
 	config.BindEnvAndSetDefault("runtime_security_config.syscall_monitor.enabled", false)
+	config.BindEnvAndSetDefault("runtime_security_config.perf_buffer_monitor.enabled", false)
```

Comment:
Could you please open a PR to the public helm chart so that we have these settings added. Thank you.

Reply:
Will do 👍

Reply:
IMHO they should remain separate because turning on the debug logging  (via `bpf_debug`) has a drastic negative effect on performance.

Reply:
I agree with Bryce, we'd like to have this perf buffer monitor activated by default once we feel confident enough its overhead is acceptable. If that's ok with you Lee, I'll leave the `perf_buffer_monitor.enabled` parameter for now, and we can consider moving to `bpf_debug` if we end up not using it.